config.json

```diff
@@ -2357,6 +2357,23 @@
           "macros"
         ],
         "difficulty": 8
+      },
+      {
+        "slug": "pov",
+        "name": "POV",
+        "uuid": "??? Do I need to create one? How about: 7",
+        "prerequisites": [
+          "pattern-matching",
+          "case",
+          "errors",
+          "recursion",
+          "enum"
```

Comment:
This should match all of the entries of the `Zipper` exercise, and be rated more difficult since it is an added feature on top of that exercise from what I can determine.

Reply:
That seems right, will do!